mountinfo/mounted_unix.go

```diff
@@ -33,13 +32,13 @@ func mountedByStat(path string) (bool, error) {
 
 func normalizePath(path string) (realPath string, err error) {
 	if realPath, err = filepath.Abs(path); err != nil {
-		return "", fmt.Errorf("unable to get absolute path for %q: %w", path, err)
+		return "", err
 	}
 	if realPath, err = filepath.EvalSymlinks(realPath); err != nil {
-		return "", fmt.Errorf("failed to canonicalise path for %q: %w", path, err)
+		return "", err
 	}
 	if _, err := os.Stat(realPath); err != nil {
-		return "", fmt.Errorf("failed to stat target of %q: %w", path, err)
+		return "", err
```

Comment:
This part of patch _might_ actually make sense (since the errors from `os.Stat` and `filepath` are probably already wrapped).

Reply:
Also, `os.IsNotExist()` does not work for errors wrapped using `%w` (see https://go.dev/play/p/RcpXLGqwO9l), and since
 * both `filepath.EvalSymlinks` and `os.Stat` already contain path in their error message;
 * `filepath.Abs` can only return an error from `os.Getwd()` which is already descriptive enough,
this part of patch does makes sense.

Reply:
> Also, os.IsNotExist() does not work for errors wrapped using %w (see https://go.dev/play/p/RcpXLGqwO9l), and since

The reverse is also true for some errors unfortunately (and based on the discussion in https://github.com/golang/go/issues/39155, I doubt that'll change)